.github/scripts/prepare-local-server.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+mkdir keycloak-dist
+
+if [[ ( -n "$GITHUB_BASE_REF" &&  "$GITHUB_BASE_REF" == "latest" ) ]] || [[ ( -n "$QUICKSTART_BRANCH" && "$QUICKSTART_BRANCH" != "main" ) ]]; then
+  VERSION=$(grep -oPm1 "(?<=<version>)[^<]+" pom.xml)
+  echo "Using corresponding Keycloak version: $VERSION"
+  URL="https://github.com/keycloak/keycloak/releases/download/${VERSION}/keycloak-${VERSION}.tar.gz"
+else
+  echo "Downloading nightly Keycloak release"
+  URL="https://github.com/keycloak/keycloak/releases/download/nightly/keycloak-999.0.0-SNAPSHOT.tar.gz"
+fi
```

Comment:
Will need to use RHBK at some point, but don't think we can until after the GA. Unless we just upload RHBK zip somewhere temporarily?